scripts/bernstein-deploy.sh

```diff
@@ -41,24 +46,27 @@ $kubectl apply -f poll.deployment.yaml \
 -f result.service.yaml \
 -f poll.ingress.yaml \
 -f result.ingress.yaml
-# $kubectl apply -f poll.deployment.yaml -f worker.deployment.yaml -f result.deployment.yaml -f poll.service.yaml -f result.service.yaml -f poll.ingress.yaml -f result.ingress.yaml
 
 echo "[$(date)] kubectl: apply traefik conf"
 $kubectl apply -f traefik.rbac.yaml \
 -f traefik.deployment.yaml \
 -f traefik.service.yaml
-# $kubectl apply -f traefik.rbac.yaml -f traefik.deployment.yaml -f traefik.service.yaml
 
-exit
+echo "[$(date)] kubectl: create postgres table"
+pg_deploy_id_pattern="kube-deploy-postgres-*-*"
+pg_deploy_id=$($kubectl get pods -o json | jq -r '.items[]|select(.metadata.name|test("$pg_deploy_id_pattern")).metadata.name')
+psql_username=username
+psql_password=password
+
+#echo "$pg_deploy_id_pattern"
+#echo "$pg_deploy_id"
+#echo "$psql_username"
+#echo "$psql_password"
 
-echo "CREATE TABLE votes \
-(id text PRIMARY KEY, vote text NOT NULL);" \
-| $kubectl exec -i <postgres-deployment-id> -c <postgres-container-id> \
-- psql -U username -P password
-# echo "CREATE TABLE votes(id text PRIMARY KEY, vote text NOT NULL);" | $kubectl exec -i <postgres-deployment-id> -c <postgres-container-id> - psql -U <username>
+echo "CREATE TABLE votes(id text PRIMARY KEY, vote text NOT NULL);" \
```

Comment:
creating a table should only be done if the table does not exists:
```suggestion
echo "CREATE TABLE IF NOT EXISTS votes(id text PRIMARY KEY, vote text NOT NULL);" \
```

Reply:
Good idea